src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -180,7 +181,7 @@ public void openFiles(List<Path> filesToOpen) {
     /**
      * This is the real file opening. Should be called via {@link #openFile(Path)}
      *
-     * Similar method: {@link org.jabref.gui.JabRefFrame#addTab(org.jabref.model.database.BibDatabaseContext, boolean)}.
```

Comment:
Please do not remove the absolute package for JavaDoc comments. I think, checkstyle will complain or some other checker?